coil-base/src/main/java/coil/memory/ViewTargetRequestManager.kt

```diff
@@ -30,14 +31,17 @@ internal class ViewTargetRequestManager : View.OnAttachStateChangeListener {
     @Volatile var currentRequestJob: Job? = null
         private set
 
+    // Metadata of the successful request attached to this view.
+    @Volatile var metadata: Metadata? = null
```

Comment:
Caching the metadata on the view so it can be grabbed with `imageView.metadata.cacheKey`.